app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -95,6 +96,21 @@ trait BucketRoutes extends Authentication
                   }
                 }
               }
+            } ~
+            authenticate { user =>
+              post {
+                entity(as[BulkSceneOperation]) { bulkOp =>
+                  bulkOp.action match {
+                    case "upsert" => onSuccess(Buckets.addScenesToBucket(bulkOp.sceneIds, bucketId)) {
+                      _ => complete(StatusCodes.Created)
+                    }
+                    case "delete" => onSuccess(Buckets.deleteScenesFromBucket(bulkOp.sceneIds, bucketId)) {
+                      _ => complete(StatusCodes.NoContent)
```

Comment:
Similarly, we don't care how many rows were deleted. The completion of the query guarantees that any rows that matched are no longer there.